packages/components/table/src/table-body/events-helper.ts

```diff
@@ -143,7 +160,9 @@ function useEvents<T>(props: Partial<TableBodyProps<T>>) {
   const handleCellMouseLeave = (event) => {
     const cell = getCell(event)
     if (!cell) return
-
+    if (cell.rowSpan > 1) {
+      toggleRowClassByCell(cell.rowSpan, event, removeClass)
```

Comment:
You may need to import `addClass` and `removeClass`, otherwise an error will occur.

Reply:
OKOK

Reply:
@btea  fixed this